analysis/webservice/algorithms/doms/ResultsStorage.py

```diff
@@ -177,21 +180,43 @@ def __insertResult(self, execution_id, primaryId, result, batch, insertStatement
 
         dataMap = self.__buildDataMap(data_dict)
         result_id = uuid.uuid4()
-        batch.add(insertStatement, (
-            result_id,
-            execution_id,
-            result["id"],
-            primaryId,
-            result["lon"],
-            result["lat"],
-            result["source"],
-            result["time"],
-            result["platform"] if "platform" in result else None,
-            result["device"] if "device" in result else None,
-            dataMap,
-            1 if primaryId is None else 0,
-            result["depth"]
-        ))
+
+        try:
```

Comment:
I'm wondering if something like this would be cleaner to avoid this try/catch and the many lines of log.error:

```python3
insert_params = (
  result_id,
  execution_id,
  result["id"],
  primaryId,
  result["lon"],
  result["lat"],
  ...
)

...
self._log.info('Inserting %s', insert_params)
batch.add(insertStatement, insert_params)
```

Reply:
My thinking was to only log failed inserts cause there can be quite a lot of them (one for every primary + secondary point &c)

Reply:
I tried cleaning up the logging &c a bit.